subgraphs/uniswap-v3-forks/protocols/sushiswap-v3/config/deployments/sushiswap-v3-arbitrum/configurations.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  Address,
+  BigDecimal,
+  BigInt,
+  Bytes,
+  log,
+} from "@graphprotocol/graph-ts";
+import { Factory } from "../../../../../generated/Factory/Factory";
+import {
+  BIGDECIMAL_ONE,
+  BIGDECIMAL_ZERO,
+  FeeSwitch,
+  Network,
+  RewardIntervalType,
+} from "../../../../../src/common/constants";
+import { Configurations } from "../../../../../configurations/configurations/interface";
+import { PROTOCOL_NAME, PROTOCOL_SLUG } from "../../../src/common/constants";
+import { stringToBytesList } from "../../../../../src/common/utils/utils";
+
+export class SushiswapV3ArbitrumConfigurations implements Configurations {
+  getNetwork(): string {
+    return Network.ARBITRUM_ONE;
+  }
+  getProtocolName(): string {
+    return PROTOCOL_NAME;
+  }
+  getProtocolSlug(): string {
+    return PROTOCOL_SLUG;
+  }
+  getFactoryAddress(): Bytes {
+    return Bytes.fromHexString("0x1af415a1eba07a4986a52b6f2e7de7003d82231e");
+  }
+  getFactoryContract(): Factory {
+    return Factory.bind(
+      Address.fromString("0x1af415a1eba07a4986a52b6f2e7de7003d82231e")
+    );
+  }
+  getProtocolFeeOnOff(): string {
+    return FeeSwitch.OFF;
+  }
+  getInitialProtocolFeeProportion(fee: i64): BigDecimal {
+    log.warning("getProtocolFeeRatio is not implemented: {}", [fee.toString()]);
+    return BIGDECIMAL_ZERO;
+  }
+  getProtocolFeeProportion(protocolFee: BigInt): BigDecimal {
+    return BIGDECIMAL_ONE.div(protocolFee.toBigDecimal());
+  }
+  getRewardIntervalType(): string {
+    return RewardIntervalType.NONE;
+  }
+  getReferenceToken(): Bytes {
```

Comment:
Would you leave a comment next to the reference token indicating which token it is with the symbol like the others?

Reply:
https://github.com/messari/subgraphs/pull/2151/commits/fa66f9ce0e791be71b2d26b1c56d28c7512b0d38